test/gpu/ops_test.cpp

```diff
@@ -154,17 +161,22 @@ void run_verify_program()
     auto cpu_arg_f = detach_async([&] { return run_cpu<V>(cpu_prog); });
     auto gpu_arg   = run_gpu<V>(gpu_prog);
     auto cpu_arg   = cpu_arg_f.get();
-    bool passed    = verify_args(migraphx::get_type_name<V>(), cpu_arg, gpu_arg);
-    if(not passed)
-    {
-        V v;
-        auto p = v.create_program();
-        std::cout << p << std::endl;
-        std::cout << "cpu:\n" << cpu_prog << std::endl;
-        std::cout << "gpu:\n" << gpu_prog << std::endl;
-        std::cout << std::endl;
+
+    std::size_t num = cpu_arg.size();
+    for(std::size_t i = 0; i < num; ++i)
+    {
+        bool passed = verify_args(migraphx::get_type_name<V>(), cpu_arg[i], gpu_arg[i]);
+        if(not passed)
+        {
+            V v;
+            auto p = v.create_program();
```

Comment:
This should not be inside the loop. There should be a loop to set the `passed` variable and then this can be called.

Reply:
fixed